### DISCUSSION TITLE
Maximum index element

### DISCUSSION THREAD

**Discussion by elyes03** [Consensys/gnark#1115] [Q&A]
Hello,

I am trying to implement logistic regression in gnark to classify images. The goal is to output the index of the weight tab that produces the maximum positive product between the image and the weights.
I tried to do it this way : 
v = api.Select(cmp16bit.IsLess(prod, prodmax), v, frontend.Variable(i))
where prodmax is the maximum product, prod is the product of the current iteration, and i is the index of the current iteration. 
This causes an error and I think it is because I assign the value v to itself inside the select.
Does someone please have an idea on how to deal with this case ?

Thank you very much.

**Comment by gbotrel** [comment]
Hi -- 
>  I assign the value v to itself inside the select.
this is not an issue;

can you post a snippet to reproduce / the actual error ?
